lib/php/Converter.php

```diff
@@ -1438,6 +1438,9 @@ public function convertNetworkInventory(array $data): array
                                 [$netport['connections']['connection']];
                         }
                         if (isset($netport['aggregate'])) {
+                            if (!is_array($netport['aggregate']['port'])) {
+                                $netport['aggregate']['port'] = [$netport['aggregate']['port']];
+                            }
                             $netport['aggregate'] = array_is_list($netport['aggregate']['port']) ?
```

Comment:
It may be simplier to do
```suggestion
                            $netport['aggregate'] = is_array($netport['aggregate']['port']) && array_is_list($netport['aggregate']['port']) ?
```

Reply:
True.